### PR TITLE
[Import Automation] Schedule import cron jobs on Cloud Scheduler, locally

### PR DESCRIPTION
**Pull request by jehangiramjad**

### CODE REVIEW

File: import-automation/executor/schedule_update_import.py [hunk 1]
```diff
@@ -62,20 +93,19 @@ def _override_configs(filename: str,
...
     )
 
     bucket = storage.Client(project_id).bucket(bucket_name,
                                                user_project=project_id)
-    blob = bucket.blob(fname)
+    blob = bucket.blob(filename)
     config_dict = json.loads(blob.download_as_string(client=None))
-
+    return config_dict
     return configs.ExecutorConfig(**config_dict['configs'])
```
Comment (ajaits) [inline import-automation/executor/schedule_update_import.py:109]: delete this line?

**Review by ajaits — APPROVED** [review]

File: import-automation/executor/README.md [hunk 2]
```diff
@@ -57,21 +57,58 @@ Once the script runs to completion, the data directory's latest update is printe
...
+Note: any local changes to the `` file are ignored by git. This was done using:
+
 ```
-Run `./schedule_update_import.sh -u <config_project_id> <path_to_import>`
+Run git update-index --skip-worktree <repo_root>/import-automation/executor/config_override.json
 ```
 
+To start tracking changes to this file, execute the following:
+```
+Run git update-index --no-skip-worktree default_values.txt <repo_root>/import-automation/executor/config_override.json
```
[the remark this answers is not in the document]
Comment (jehangiramjad) [reply import-automation/executor/README.md]: Sorry, removed!

File: import-automation/executor/README.md [hunk 3]
```diff
@@ -57,21 +57,58 @@ Once the script runs to completion, the data directory's latest update is printe
...
+```
+Run git ls-files -v . | grep ^S
+```
+
+### Usage
+
 Run `./schedule_update_import.sh --help` for usage.
 
 `<config_project_id>` is the GCP project id where the config file is stored, e.g. `datcom-import-automation`.
 `<path_to_import>` is the path to the import (relative to the root directory of the `data` repo), with the name of the import provided with a colon, e.g. `scripts/us_usda/quickstats:UsdaAgSurvey`.
```
[the remark this answers is not in the document]
Comment (jehangiramjad) [reply import-automation/executor/README.md]: Done, thanks!

File: import-automation/executor/schedule_update_import.py [hunk 4]
```diff
@@ -51,6 +56,32 @@
 logging.basicConfig(level=logging.INFO)
 
 
+def _get_cron_schedule(repo_dir: str, absolute_import_path: str,
+                       manifest_filename: str):
+
+    path = absolute_import_path
+    if ":" in absolute_import_path:
+        path = absolute_import_path.split(":")[0]
```
[the remark this answers is not in the document]
Comment (jehangiramjad) [reply import-automation/executor/schedule_update_import.py]: Done!

File: import-automation/executor/schedule_update_import.py [hunk 5]
```diff
@@ -51,6 +56,32 @@
...
+    if not os.path.isfile(manifest_fp):
+        raise Exception(
+            f'Manifest for import could not be found. Import: {absolute_import_path}. Looking for {manifest_filename} at path: {manifest_fp}'
+        )
+
+    manifest = import_executor.parse_manifest(manifest_fp)
+    validation.is_manifest_valid(manifest, repo_dir, path)
+
+    for spec in manifest['import_specifications']:
+        if spec['import_name'] in absolute_import_path:
```
[the remark this answers is not in the document]
Comment (jehangiramjad) [reply import-automation/executor/schedule_update_import.py]: Nice suggestion. Done!

File: import-automation/executor/schedule_update_import.py (hunk 1 above)
Comment (jehangiramjad) [reply import-automation/executor/schedule_update_import.py]: Done

File: import-automation/executor/app/executor/cloud_scheduler.py [hunk 6]
```diff
@@ -110,6 +126,8 @@ def create_or_update_job(project_id, location: str, job: Dict) -> Dict:
     job['name'] = f'{parent}/jobs/{job["name"]}'
 
     try:
+        print(parent)
+        print(job)
```
[the remark this answers is not in the document]
Comment (jehangiramjad) [reply import-automation/executor/app/executor/cloud_scheduler.py]: Removed

**Pull request merged**